### DISCUSSION TITLE
Vercel is attempting to link to a github account that is in no related to the account

### DISCUSSION THREAD

**Discussion by mike-falisz** [vercel/community#2107] [Help] [answered]
### Summary

I can not switch the account to log-in/connect to the correct account. Has anyone experienced this? WOuld love some help undoing from either my or the account admin side.

**Comment by amyegan** [comment, marked as answer]
Hi @mike-falisz! Please email [registration@vercel.com](mailto:registration@vercel.com) and include your Vercel and GitHub usernames. I can't do much from here since your private account information would be put out in the open. But the Customer Success Team can work with you privately via email

**Comment by mike-falisz** [comment]
Hi Amy! Thanks for the quick response. I was able to figure it out. Everything seemed correct on my dashboard so I closed the pr that was sticking, reopened and it ran the check with no problem.
